imap/reconstruct.c

```diff
@@ -159,7 +157,35 @@ int main(int argc, char **argv)
 
     progname = basename(argv[0]);
 
-    while ((opt = getopt(argc, argv, "C:p:rmfsxGqRUMIoOnV:uP")) != EOF) {
+    /* keep this in alphabetical order */
+    static const char *const short_options = "C:GIMOPRUV:fmnop:qrsux";
+
+    static const struct option long_options[] = {
+        /* n.b. no long option for -C */
+        { "force-reparse", no_argument, NULL, 'G' },
+        { "update-uniqueids", no_argument, NULL, 'I' },
+        { "prefer-mboxlist", no_argument, NULL, 'M' },
+        { "delete-odd-files", no_argument, NULL, 'O' },
+        { "header-paths", no_argument, NULL, 'P' },
+        { "guid-mismatch-keep", no_argument, NULL, 'R' },
+        { "guid-mismatch-discard", no_argument, NULL, 'U' },
+        { "set-version", required_argument, NULL, 'V' },
+        { "scan-filesystem", no_argument, NULL, 'f' },
+        { "dry-run", no_argument, NULL, 'n' },
```

Comment:
I might have missed the explanation: why does the `m` option not have a long option? Is that for the same reason it's not mentioned in the documentation?

Reply:
Huh, I guess I just missed it -- probably because it's not documented.  I'll figure out what it does, and then do something with it.  Thanks!

Reply:
Oh, `-m` is this thing: #3068 -- it's a long-dead option, which we haven't yet quite decided whether to just get rid of it completely, or rewrite the long-missing functionality.

There's no documentation anymore cause that was cleaned up in 56b067a1d65852e7116eb717e94866bb49c0b35f

I'm thinking I'll leave it alone in this PR.  Some future PR will either bring it back properly, with a suitable long option, or discard the remnants forever.